tests/javaRosa.js

```diff
@@ -785,6 +785,7 @@ require([
                     <translation lang="en" default="">\
                         <text id="question1-label">\
                             <value>question1 en label</value>\
+                            <value form="markdown">question1 en help</value>\
```

Comment:
I misread the spec. Confirmed with Ro and will. Should be in question1-help now.